examples/web-worker/pages/index.vue

```diff
@@ -82,7 +82,7 @@ export default {
       if (worker) worker.postMessage({ hello: 'world' })
       else this.notification = 'No more test workers available'
     },
-    long (miliseconds) {
+    long (milliseconds) {
```

Comment:
lol why are there spaces after method names?

Reply:
Yep, can be removed

Reply:
ESLint @qm3ster

Reply:
Opps, I need more sleeping now 🤕